core/src/main/java/org/opensearch/sql/expression/function/OpenSearchFunction.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.expression.function;
+
+import static org.opensearch.sql.data.type.ExprCoreType.BOOLEAN;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.Getter;
+import lombok.Setter;
+import org.opensearch.sql.data.model.ExprValue;
+import org.opensearch.sql.data.type.ExprType;
+import org.opensearch.sql.expression.Expression;
+import org.opensearch.sql.expression.FunctionExpression;
+import org.opensearch.sql.expression.NamedArgumentExpression;
+import org.opensearch.sql.expression.env.Environment;
+
+/**
+ * OpenSearch functions are FunctionExpression objects used by the core analyzer TODO: move to the
```

Comment:
Put `<br`> before TODO. Update TODO to mention that will be the next phase of refactor (not an imcomplete work there)